build.gradle

```diff
@@ -75,6 +75,7 @@ dependencies {
     testImplementation 'com.google.android:support-v4:r6'
     testImplementation 'com.google.android.gms:play-services-ads:18.3.0'
     testImplementation 'com.google.android.gms:play-services-base:17.1.0'
+    testImplementation 'org.json:json:20140107'
```

Comment:
https://stackoverflow.com/questions/29402155/android-unit-test-not-mocked

Reply:
I believe we do use JSONObject in other test files, why this time there would be errors?

Reply:
Could you try running these tests locally @qingzhuozhen ? This is one of the issues we may solve with multiple devices

Reply:
yeah, I can run the tests locally. I guess the issue is for in the test case `testDoNotSendAppSetId` we use PowerMock there. I wonder if we should adjust our code a bit so we don't need to use PowerMock there and just use Mockito, then we don't really need to import the lib into our tests.